### ISSUE TITLE
is it impossible to use special character like `$`?

### ISSUE THREAD

**Issue by mixin27** [petercinibulk/envied#121]
I have a problem while using environment value like `!21ADcsed$tY?}{`.

`.env`
```.env
API_KEY="!21ADcsed$tY?}{"
```
After running `build_runner`, I've got the following result.

```dart
static const String apiKey = '!21ADcsed?}{';
```

It does not include `$tY`.

I want to know, is it impossible to  use this character or can you support for using this character?

Thanks

**Comment by jpeiffer** [comment]
Try adding `rawString: true` to the `@EnviedField` annotation.  Looking through the existing tests, that seems to be what is required to make certain special characters work.

**Comment by mixin27** [comment]
Thank you for your support @jpeiffer but it doesn't work because `rawString: true` just generate a raw string instead of `String` like `r'!21ADcsed?}{'` when I tested. I am not sure about that.

Luckily, I've got a solution. I added `escape` character before `$`. This will generate like this:
```dart
 static const String apiKey = '!21ADcsed$tY?}{';
```

But it does not work while compiling, so I added another `escape` character like this.

```env
API_KEY="!21ADcsed\\$tY?}{"
```

So the result will be:
```dart
static const String apiKey = '!21ADcsed\$tY?}{';
```

which is what I need.
